draft-ietf-quic-transport.md

```diff
@@ -2491,13 +2495,11 @@ connection in a recoverable state, the endpoint can send a RST_STREAM frame
 ({{frame-rst-stream}}) with an appropriate error code to terminate just the
 affected stream.
 
-Other than STOPPING ({{solicited-state-transitions}}), RST_STREAM MUST be
-instigated by the application and MUST carry an application error code.
-Resetting a stream without knowledge of the application protocol could cause the
-protocol to enter an unrecoverable state.  Application protocols might require
-certain streams to be reliably delivered in order to guarantee consistent state
-between endpoints.
-
+RST_STREAM MUST be instigated by the application.  RST_STREAM carries an
```

Comment:
Along with the reference to STOPPING, you've removed the ability for the QUIC stack to handle STOP_SENDING internally.  I don't think that's an improvement, and perhaps not intended here.

Reply:
Copying the code from STOP_SENDING works, doesn't it?

Reply:
"RST_STREAM MUST be instigated by the application."  That means the QUIC stack never initiates a RST, which is unrelated to what error code you put in the RST_STREAM that it's not initiating.

Perhaps we say, "Unless in response to a STOP_SENDING ({{solicited...}}), RST_STREAM...."?